app/controllers/work_move_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# a controller for changing the collection of work.
+class WorkMoveController < ApplicationController
+  before_action :authenticate_user!
+  verify_authorized except: %i[search]
```

Comment:
Can you add a comment here about this decision. "We don't authorize on the search action because ..."

Reply:
I just removed the exception. There was no good reason for it.